vnext/Microsoft.ReactNative/Views/ViewManagerBase.h

```diff
@@ -92,6 +92,8 @@ class REACTWINDOWS_EXPORT ViewManagerBase : public IViewManager {
 
   virtual void TransferProperties(const XamlView &oldView, const XamlView &newView);
 
+  virtual void OnViewUnloaded(XamlView view) {}
```

Comment:
>XamlView [](http://example.com/codeflow?start=30&length=8)

Could we pass it as `const XamlView&` to avoid extra work to copy the object that causes the extra AddRef/Release calls?

Reply:
@vmoroz Yes we should be able to. I was mocking the syntax of the similar method [OnViewCreated](https://github.com/microsoft/react-native-windows/blob/69ba6148fd4b9bc908187b964a67633d4862bda4/vnext/Microsoft.ReactNative/Views/ViewManagerBase.h#L99) which seems to copy. Are we okay if the methods have different ways of passing the view or should OnViewCreated be changed as well?